rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -79,6 +79,9 @@ Use the Web Recording Browser to record website navigation actions. It behaves a
 * Save your attribute selection of a web element to reuse it when recording the same web elements on a website.
 * Record mouse hover positioning during web navigation.
 
+[NOTE]
+Japanese localization during Web recording is fully supported.
```

Comment:
Does this mean we support browsing Japanese websites and recording actions without issues? Or this means the Web recording UI is localized to Japanese?

Reply:
We did support browsing Japanese websites before. What was not always working before is the recording of Japanese keyboard input, e.g. such as typing Japanese characters into a search field inside a web page (aka the "Toshiba" bug). This will work now, Recorder is now able to record Japanese keyboard input on websites.